pkg/log/format.go

```diff
@@ -15,7 +22,44 @@ func getGCPFormatter() logrus.Formatter {
 }
 
 func getDevFormatter() logrus.Formatter {
-	return &logrus.TextFormatter{}
+	return devFormatter{}
+}
+
+type devFormatter struct{}
+
+func (devFormatter) Format(e *logrus.Entry) ([]byte, error) {
+	var buf bytes.Buffer
+	var sprintf func(format string, a ...interface{}) string
+	switch e.Level {
+	case logrus.DebugLevel:
+		sprintf = color.New(0xffccc).Sprintf
```

Comment:
I'm just curious what color this is

Reply:
It's light grey. Good point though @robjloranger, I'll make a constant and give it the color name :)